### DISCUSSION TITLE
What is this colored line on the side ? how do i turn it off pls .-.

### DISCUSSION THREAD

**Discussion by minhmacg** [helix-editor/helix#8485] [Q&A]
![image](https://github.com/helix-editor/helix/assets/63334432/3f9e7ed0-663f-4599-9f41-f9e4e4d63b4a)

**Comment by dusty-phillips** [comment]
It’s your version control diff. Disable it by removing `diff` from 1st editor.gutters`.

https://docs.helix-editor.com/configuration.html#editor

**Reply by minhmacg** [reply]
Thank you for the fast reply
